x/evmutil/keeper/conversion_evm_native_bep3_test.go

```diff
@@ -0,0 +1,415 @@
+package keeper_test
+
+import (
+	"fmt"
+	"testing"
+
+	sdkmath "cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/kava-labs/kava/x/evmutil/testutil"
+	"github.com/kava-labs/kava/x/evmutil/types"
+)
+
+type Bep3ConversionTestSuite struct {
+	testutil.Suite
+}
+
+var (
+	bep3Denoms = []string{"bnb", "busd", "btcb", "xrpb"}
+)
+
+func TestBep3ConversionTestSuite(t *testing.T) {
+	suite.Run(t, new(Bep3ConversionTestSuite))
+}
+
+func (suite *Bep3ConversionTestSuite) TestConvertCoinToERC20_Bep3() {
+	for _, denom := range bep3Denoms {
```

Comment:
This actually makes the tests quite a bit easier to read -- easy to see that all denoms are tested and much more resistant to refactorings that could drop a denom, accidentally rename one, break the check some other unique way, etc